src/Data/ProgrammesDb/Entity/ProgrammeItem.php

```diff
@@ -224,4 +224,11 @@ public function setSegmentEventCount(int $segmentEventCount): void
     {
         $this->segmentEventCount = $segmentEventCount;
     }
+
+    public function clearVersionAssociations(): void
```

Comment:
I assume this is only to be used in special circumstances. Should there be a warning comment on it?